File: src/main/java/classifications/CharacterRace.java
package classifications;

import java.util.List;

public class CharacterRace {
//The race gives the character an armor class, a list of racial traits, a base weapon, Weapon and/or spell proficiencies, and potentially a subrace
    protected String name;
    protected String description;
    protected int armorClass;


    public CharacterRace(String name, String description) {
        this.name = name;
        this.description = description;
    }

    public String getName() {
        return name;
    }
}

File: src/main/kotlin/idiotamspielen/vttproject/classifications/CharacterRace.kt
package idiotamspielen.vttproject.classifications
/*The race gives the character an armor class, a list of racial traits, a base weapon,
 * Weapon and/or spell proficiencies, and potentially a subrace
 */
open class CharacterRace(
    protected var description: String?
) : Nameable {
    protected var armorClass: Int = 0
    override fun getName(): String {
        TODO("Not yet implemented")
    }
}
